diff --git a/app/src/schema.rs b/app/src/schema.rs
index fd3aaf7..a5bdd91 100644
--- a/app/src/schema.rs
+++ b/app/src/schema.rs
@@ -1,31 +1,31 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     minutes_per_pings (id) {
         id -> Integer,
         minutes_per_ping -> Integer,
-        timestamp -> Timestamp,
+        timestamp -> TimestamptzSqlite,
         counter -> Integer,
         node -> Integer,
     }
 }
 
 diesel::table! {
     pings (id) {
         id -> Integer,
-        ping -> Timestamp,
+        ping -> TimestamptzSqlite,
     }
 }
 
 diesel::table! {
     tags (id) {
         id -> Integer,
-        ping -> Timestamp,
+        ping -> TimestamptzSqlite,
         tag -> Nullable<Text>,
-        timestamp -> Timestamp,
+        timestamp -> TimestamptzSqlite,
         counter -> Integer,
         node -> Integer,
     }
 }
 
 diesel::allow_tables_to_appear_in_same_query!(
